jvc_projector/__init__.py

```diff
@@ -112,12 +124,15 @@ def command(self, command_string):
             self._send_command(Commands[command_string].value)
             return True
 
-    def is_on(self):
-        message = self._send_command(Commands.power_status.value, ack = ACKs.power_ack.value)
-        if message == PowerStates.lamp_on.value or message == PowerStates.reserved.value:
-            return True
-        elif message == PowerStates.standby.value or message == PowerStates.cooling.value or message == PowerStates.emergency.value:
-            return False
+    def power_state(self):
+        message = self._send_command(Commands.power_status.value, ack=ACKs.power_ack.value)
+        for x in PowerStates:
```

Comment:
[See here](https://docs.python.org/3/library/enum.html#programmatic-access-to-enumeration-members-and-their-attributes). You should be able to do `return PowerStates(message).name` instead of the `for` loop.